### PR TITLE
manually create cdn distros for custom domains

### PR DESCRIPTION
**Pull request by spgreenberg**

### CODE REVIEW

File: manual-cdn/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## Creating a new CDN
+
+Certificate generation requires real-time coordination with the customer's DNS admin. DNS challenges must be configured and verified in order to proceed. 
+
+Run the `create-distribution.sh` script with the above inputs.  The script will do the following:
+
+- Set up the correct directories for each domain in `$base-directory`.
+- Run certbot
+- Generate a *.tfvars file in `$base-directory/$domain`
```
Comment (mxplusb) [inline manual-cdn/README.md]:
```suggestion
- Generate a `*.tfvars` file in `$base-directory/$domain`
```

**Comment by pburkholder** [comment]
Is this an opportunity for us to have a Security Considerations discussion?

**Comment by spgreenberg** [comment]
@pburkholder Sure. I am not sure what you mean but happy to talk it through.

**Comment by pburkholder** [comment]
@spgreenberg There are a lot changes to GoLang files, so if the PR is primarily a documentation change let's disentangle that. If it is a significant code change, then let's be sure to make comments on the potential implications of those changes.

As I'm rusty w/ go (aka, I hardly know the language :/) I can't help much on that end ATM. P.

**Comment by bengerman13** [comment]
@spgreenberg what's the status on this? Is it really ready for review, or stale or...?

**Comment by spgreenberg** [comment]
hoping I can remove this. need it around for a few more days to test manual domain migrations.

**Comment by bengerman13** [comment]
We can nuke this PR, right?

**Comment by spgreenberg** [comment]
yes

**Pull request closed without merging**